triple-store/triple-store-impl-jena/src/main/java/com/powsybl/triplestore/impl/jena/TripleStoreJena.java

```diff
@@ -152,6 +165,32 @@ public PropertyBags query(String query) {
         return results;
     }
 
+    @Override
+    public void copyFrom(TripleStore source) {
+        Dataset sourceDataset = ((TripleStoreJena) source).getDataset();
+        Iterator<String> names = sourceDataset.listNames();
+        while (names.hasNext()) {
```

Comment:
You should use a for-loop instead of a while here as you iterate over all the names:
```
for (String name : sourceDataset.listNames()) {
}
```

Reply:
Done